[2023-10-28 00:20:21,523] DEBUG add_fornecedor() L47   Adicionado fornecedor de nome: 'Deposito Bebidas Brasil' - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L47  
[2023-10-28 00:20:21,538] INFO _log() L224  127.0.0.1 - - [28/Oct/2023 00:20:21] "POST /fornecedor HTTP/1.1" 200 - - call_trace=C:\Users\lab\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2023-10-28 00:21:11,280] DEBUG add_fornecedor() L39   Adicionando fornecedor de nome: 'Deposito Bebidas RJ' - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L39  
[2023-10-28 00:21:11,300] DEBUG add_fornecedor() L47   Adicionado fornecedor de nome: 'Deposito Bebidas RJ' - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L47  
[2023-10-28 00:21:11,304] INFO _log() L224  127.0.0.1 - - [28/Oct/2023 00:21:11] "POST /fornecedor HTTP/1.1" 200 - - call_trace=C:\Users\lab\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2023-10-28 00:21:22,862] DEBUG add_fornecedor() L39   Adicionando fornecedor de nome: 'Deposito Bebidas SP' - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L39  
[2023-10-28 00:21:22,876] DEBUG add_fornecedor() L47   Adicionado fornecedor de nome: 'Deposito Bebidas SP' - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L47  
[2023-10-28 00:21:22,876] INFO _log() L224  127.0.0.1 - - [28/Oct/2023 00:21:22] "POST /fornecedor HTTP/1.1" 200 - - call_trace=C:\Users\lab\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2023-10-28 00:22:09,789] DEBUG add_fornecedor() L39   Adicionando fornecedor de nome: 'So Refri SP' - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L39  
[2023-10-28 00:22:09,812] DEBUG add_fornecedor() L47   Adicionado fornecedor de nome: 'So Refri SP' - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L47  
[2023-10-28 00:22:09,814] INFO _log() L224  127.0.0.1 - - [28/Oct/2023 00:22:09] "POST /fornecedor HTTP/1.1" 200 - - call_trace=C:\Users\lab\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2023-10-28 00:22:55,787] DEBUG get_fornecedores() L70   Coletando fornecedores  - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L70  
[2023-10-28 00:22:55,791] DEBUG get_fornecedores() L80   4 fornecedores encontrados - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L80  
[2023-10-28 00:22:55,795] INFO _log() L224  127.0.0.1 - - [28/Oct/2023 00:22:55] "GET /fornecedores HTTP/1.1" 200 - - call_trace=C:\Users\lab\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2023-10-28 00:24:02,076] ERROR log_exception() L1449 Exception on /fornecedor [GET] - call_trace=C:\Users\lab\anaconda3\Lib\site-packages\flask\app.py L1449
Traceback (most recent call last):
  File "C:\Users\lab\anaconda3\Lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\lab\anaconda3\Lib\site-packages\flask\app.py", line 1519, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\lab\anaconda3\Lib\site-packages\flask_cors\extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ^^^^^^^^^^^^^^^^^^
  File "C:\Users\lab\anaconda3\Lib\site-packages\flask\app.py", line 1517, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\lab\anaconda3\Lib\site-packages\flask\app.py", line 1503, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\lab\anaconda3\Lib\site-packages\flask_openapi3\scaffold.py", line 109, in wrapper
    resp = _do_wrapper(
           ^^^^^^^^^^^^
  File "C:\Users\lab\anaconda3\Lib\site-packages\flask_openapi3\do_wrapper.py", line 135, in _do_wrapper
    response = func(**request_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\dev\pyprojects\devfull-mvp1\app_api\app.py", line 93, in get_fornecedor
    fornecedor_id = query.id
                    ^^^^^^^^
AttributeError: 'FornecedorBuscaSchema' object has no attribute 'id'
[2023-10-28 00:24:02,105] INFO _log() L224  127.0.0.1 - - [28/Oct/2023 00:24:02] "[35m[1mGET /fornecedor?nome=So%20Refri%20SP HTTP/1.1[0m" 500 - - call_trace=C:\Users\lab\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2023-10-28 00:27:31,464] INFO _log() L224  127.0.0.1 - - [28/Oct/2023 00:27:31] "[32mGET / HTTP/1.1[0m" 302 - - call_trace=C:\Users\lab\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2023-10-28 00:27:31,473] INFO _log() L224  127.0.0.1 - - [28/Oct/2023 00:27:31] "GET /openapi/ HTTP/1.1" 200 - - call_trace=C:\Users\lab\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2023-10-28 00:27:31,517] INFO _log() L224  127.0.0.1 - - [28/Oct/2023 00:27:31] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\lab\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2023-10-28 00:27:31,524] INFO _log() L224  127.0.0.1 - - [28/Oct/2023 00:27:31] "[36mGET /openapi/static/images/redoc.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\lab\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2023-10-28 00:27:31,526] INFO _log() L224  127.0.0.1 - - [28/Oct/2023 00:27:31] "[36mGET /openapi/static/images/rapidoc.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\lab\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2023-10-28 00:27:41,184] INFO _log() L224  127.0.0.1 - - [28/Oct/2023 00:27:41] "[33mGET /index.html HTTP/1.1[0m" 404 - - call_trace=C:\Users\lab\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2023-10-28 00:27:41,241] INFO _log() L224  127.0.0.1 - - [28/Oct/2023 00:27:41] "[33mGET /favicon.ico HTTP/1.1[0m" 404 - - call_trace=C:\Users\lab\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2023-10-28 00:30:02,442] INFO _log() L224  127.0.0.1 - - [28/Oct/2023 00:30:02] "[33mGET /index.html HTTP/1.1[0m" 404 - - call_trace=C:\Users\lab\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2023-10-28 00:30:02,534] INFO _log() L224  127.0.0.1 - - [28/Oct/2023 00:30:02] "[33mGET /favicon.ico HTTP/1.1[0m" 404 - - call_trace=C:\Users\lab\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2023-10-28 00:30:17,470] INFO _log() L224  127.0.0.1 - - [28/Oct/2023 00:30:17] "[33mGET /index.html HTTP/1.1[0m" 404 - - call_trace=C:\Users\lab\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2023-10-28 00:32:46,911] INFO _log() L224  127.0.0.1 - - [28/Oct/2023 00:32:46] "[33mGET /app_front/index.html HTTP/1.1[0m" 404 - - call_trace=C:\Users\lab\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2023-10-28 00:33:06,841] INFO _log() L224  127.0.0.1 - - [28/Oct/2023 00:33:06] "[33mGET /index.html HTTP/1.1[0m" 404 - - call_trace=C:\Users\lab\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2023-10-28 00:35:29,329] DEBUG get_fornecedores() L70   Coletando fornecedores  - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L70  
[2023-10-28 00:35:29,335] DEBUG get_fornecedores() L80   4 fornecedores encontrados - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L80  
[2023-10-28 00:35:29,336] INFO _log() L224  127.0.0.1 - - [28/Oct/2023 00:35:29] "GET /fornecedores HTTP/1.1" 200 - - call_trace=C:\Users\lab\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2023-10-28 00:35:44,448] INFO _log() L224  127.0.0.1 - - [28/Oct/2023 00:35:44] "OPTIONS /fornecedor?nome=So%20Refri%20SP HTTP/1.1" 200 - - call_trace=C:\Users\lab\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2023-10-28 00:35:44,454] DEBUG del_produto() L120  Deletando dados sobre produto #So Refri SP - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L120 
[2023-10-28 00:35:44,469] DEBUG del_produto() L129  Deletado produto #So Refri SP - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L129 
[2023-10-28 00:35:44,470] INFO _log() L224  127.0.0.1 - - [28/Oct/2023 00:35:44] "DELETE /fornecedor?nome=So%20Refri%20SP HTTP/1.1" 200 - - call_trace=C:\Users\lab\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2023-10-28 00:36:48,607] DEBUG get_fornecedores() L70   Coletando fornecedores  - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L70  
[2023-10-28 00:36:48,611] DEBUG get_fornecedores() L80   3 fornecedores encontrados - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L80  
[2023-10-28 00:36:48,613] INFO _log() L224  127.0.0.1 - - [28/Oct/2023 00:36:48] "GET /fornecedores HTTP/1.1" 200 - - call_trace=C:\Users\lab\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2023-10-28 00:37:39,943] DEBUG get_fornecedores() L70   Coletando fornecedores  - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L70  
[2023-10-28 00:37:39,946] DEBUG get_fornecedores() L80   3 fornecedores encontrados - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L80  
[2023-10-28 00:37:39,947] INFO _log() L224  127.0.0.1 - - [28/Oct/2023 00:37:39] "GET /fornecedores HTTP/1.1" 200 - - call_trace=C:\Users\lab\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2023-10-28 00:37:41,667] DEBUG get_fornecedores() L70   Coletando fornecedores  - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L70  
[2023-10-28 00:37:41,668] DEBUG get_fornecedores() L80   3 fornecedores encontrados - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L80  
[2023-10-28 00:37:41,670] INFO _log() L224  127.0.0.1 - - [28/Oct/2023 00:37:41] "GET /fornecedores HTTP/1.1" 200 - - call_trace=C:\Users\lab\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2023-10-28 00:37:43,393] DEBUG get_fornecedores() L70   Coletando fornecedores  - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L70  
